docs/tfengine/schemas/resources.md

```diff
@@ -752,6 +752,54 @@ Type: array(string)
 
 Type: array()
 
+### healthcare_datasets.consent_stores
+
+consent stores to create.
+
+Type: array(object)
+
+### healthcare_datasets.consent_stores.iam_members
+
+IAM member to grant access for.
+
+Type: array(object)
+
+### healthcare_datasets.consent_stores.iam_members.member
+
+Member to grant acess to role.
+
+Type: string
+
+### healthcare_datasets.consent_stores.iam_members.role
+
+IAM role to grant.
+
+Type: string
+
+### healthcare_datasets.consent_stores.labels
+
+Labels to set on the consent store.
```

Comment:
Could you link some documentation on what these labels are? It seems like just a key-value pair but the documentation just says it's "object" and it's not obvious how it's structured.